modules/serverless-install-kafka-yaml.adoc

```diff
@@ -0,0 +1,81 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-install-kafka-yaml_{context}"]
+= Installing Apache Kafka components using YAML
+
+Cluster administrators can enable the use of Apache Kafka functionality in a {ServerlessProductName} deployment by instantiating the KnativeKafka custom resource definition provided by the *Knative Kafka* {ServerlessOperatorName} API.
+
+.Prerequisites
+* You must have access to an AMQ Streams cluster.
+* You must have installed the {ServerlessOperatorName} and Knative Eventing.
+* You must have cluster administrator permissions on the {product-title} cluster.
+
+.Procedure
+
+. Create and apply a `.yaml` file that contains the following:
++
+[source,yaml]
+----
+apiVersion: operator.serverless.openshift.io/v1alpha1
+kind: KnativeKafka
+metadata:
+    name: knative-kafka
+    namespace: knative-eventing
+spec:
+    channel:
+        enabled: true <1>
+        bootstrapServers: <bootstap_server> <2>
+    source:
+        enabled: true <3>
+----
+<1> Enables developers to use the KafkaChannel channel type in the cluster.
+<2> A comma-separated list of bootstrap servers from your AMQ Streams cluster.
```

Comment:
bootstrapped?
BTW do you edit the YAML file too, there is a typo - bootstap_server